ajax/libs/kube/package.json

```diff
@@ -16,10 +16,11 @@
   "autoupdate": {
     "source": "git",
     "target": "git://github.com/imperavi/kube.git",
-    "basePath": "",
+    "basePath": "dist",
     "files": [
       "css/*",
-      "js/*"
+      "js/*",
+      "!(scss)/*"
```

Comment:
It seems a lot of libraries explicitly state what to include as opposed to what to excluse, so maybe it's better to just remove the scss line instead? That way, if some directory restructuring happens, we're more confident cdnjs won't pull in junk or too much data.